Vagrantfile

```diff
@@ -8,70 +8,12 @@
 
 
 Vagrant.configure(2) do |config|
-  # The most common configuration options are documented and commented below.
-  # For a complete reference, please see the online documentation at
-  # https://docs.vagrantup.com.
-  
-  config.vm.provider "virtualbox" do |v|
-    v.customize ["setextradata", :id, "VBoxInternal2/SharedFoldersEnableSymlinksCreate/v-root", "1"]
+    config.vm.provider "virtualbox" do |v|
+      v.customize ["setextradata", :id, "VBoxInternal2/SharedFoldersEnableSymlinksCreate/v-root", "1"]
     end
-
-  # Every Vagrant development environment requires a box. You can search for
-  # boxes at https://atlas.hashicorp.com/search.
-  config.vm.box = "hashicorp/precise32"
-
-  # Disable automatic box update checking. If you disable this, then
-  # boxes will only be checked for updates when the user runs
-  # `vagrant box outdated`. This is not recommended.
-  # config.vm.box_check_update = false
-
-  # Create a forwarded port mapping which allows access to a specific port
-  # within the machine from a port on the host machine. In the example below,
-  # accessing "localhost:8080" will access port 80 on the guest machine.
+  config.vm.provision "shell", inline: 'echo "export VAGRANT=true" >> ~/.profile'
+  config.vm.provision "shell", inline: 'apt-get install -y unzip python-pip build-essential git gfortran libatlas-base-dev python-dev && pip install -U pip'
```

Comment:
Could we instead of hard linking the packages execute `srcipts/install_prerequisites.sh` instead? Because right now we are duplicating the dependency list in there as well as here, so one of those will eventually end up out of date.

Having the prerequisites script around is handy as this can be used by any Debian based OS, including the whole Ubuntu family